shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/SafeHash.hs

```diff
@@ -172,3 +176,17 @@ data EraIndependentScriptData
 data EraIndependentPParamView
 
 data EraIndependentWitnessPPData
+
+-- ==============================================================
+-- Sometimes one wants to hash multiple things, simply by
```

Comment:
Same comment as before: comments like this are more useful as haddock on the thing in question rather than as inline comments in the module.

Reply:
done